kube/kubetypes/types.go

```diff
@@ -27,4 +27,20 @@ const (
 	MetricEgressServiceCount             = "k8s_egress_service_resources"
 	MetricProxyGroupEgressCount          = "k8s_proxygroup_egress_resources"
 	MetricProxyGroupIngressCount         = "k8s_proxygroup_ingress_resources"
+
+	// TODO: rename this file, this is not a metric
+	// Keys that containerboot writes to state file that can be used to determine its state.
+	// fields set in Tailscale state Secret. These are mostly used by the Tailscale Kubernetes operator to determine
+	// the state of this tailscale device.
+	KeyDeviceID   string = "device_id"   // node stable ID of the device
+	KeyDeviceFQDN string = "device_fqdn" // device's tailnet hostname
+	KeyDeviceIPs  string = "device_ips"  // device's tailnet IPs
+	// KeyCapVer contains Tailscale capability version of a proxy in form <capver>[:<pod-uid>]. This is used by the
```

Comment:
How come we don't have a separate key for the UID instead of appending it here? Naively, it feels like it would be much cleaner to have a separate key for it, but perhaps I'm missing a reason why it needs to be more atomic than that.

Reply:
The reader does need to know that the capver is for the right UID.
However, it is only written in one place, so we _could_ just write the UID first and in a separate field. That might be cleaner 👀

Reply:
I did end up writing UID in a separate field